File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/AllyHeraldKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class AllyHeraldKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playAllyHeraldKillAnimation() {
        new AnimationTester().testAnimation(new AllyHeraldKillAnimation());
    }
}

File: test/hud/animations/AllyHeraldKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {AllyHeraldKillAnimation} from '../../../src/hud/animations/AllyHeraldKillAnimation.js';

/**
 * Integration test for ally herald kill animation on actual hardware.
 */
describe('AllyHeraldKillAnimation Integration', () => {
    test('playAllyHeraldKillAnimation', async () => {
        await new AnimationTester().testAnimation(new AllyHeraldKillAnimation());
    }, 30000);
});
